WalletWasabi.Fluent/Helpers/StartupHelper.cs

```diff
@@ -62,5 +62,33 @@ private static void StartOnWindowsStartup(bool runOnSystemStartup, string pathTo
 				key.DeleteValue(nameof(WalletWasabi), false);
 			}
 		}
+
+		private static async Task StartOnLinuxStartupAsync(bool runOnSystemStartup)
+		{
```

Comment:
I think this class would be much clearer if splited into multiple ones and in this class we would have:

```csharp
class StartupHelper
{
    public static readonly LinuxStartupHelper linuxHelper = new(whatever parameters); // the instantiation would allow much easier unit testing
   // ...
}
```

Reply:
I thought you meant the Test file, will do this too.